webapp/channels/src/components/channel_invite_modal/channel_invite_modal.tsx

```diff
@@ -426,33 +425,37 @@ export default class ChannelInviteModal extends React.PureComponent<Props, State
         );
 
         return (
-            <Modal
+            <LegacyModal
```

Comment:
Do we need to make this change for other modals as Create Category or Delete Category? Error is still happening in the test server of this PR